src/Prefixer.js

```diff
@@ -34,10 +34,10 @@ export function prefix (value, length) {
 			return WEBKIT + value + replace(value, /(\w+).+(:[^]+)/, WEBKIT + 'box-$1$2' + MS + 'flex-$1$2') + value
 		// align-self
 		case 5443:
-			return WEBKIT + value + MS + 'flex-item-' + replace(value, /flex-|-self/, '') + value
+			return WEBKIT + value + MS + 'flex-item-' + replace(value, /flex-|-self/g, '') + (!match(value, /flex-|baseline/) ? MS + 'grid-row-' + replace(value, /flex-|-self/g, '') : '') + value
```

Comment:
https://github.com/thysultan/stylis/pull/285/files#r824580764

Reply:
@SukkaW Any reason why `/flex-|-self/` is converted to `/flex-|-self/g` and also `/align-content|flex-|-self/g`, does a test fail because of this, can there be multiple flex- -self notations in the declaration?

Reply:
`align-self:flex-start;` should be prefixed as `-ms-flex-line-pack:end;`. That's why `g` is added.